src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -336,12 +336,19 @@ export default class Mutations extends React.Component<
         axisMode: AxisScale
     ): JSX.Element {
         return (
-            <LollipopTooltipCountInfo
-                count={count}
-                mutations={mutations}
-                axisMode={axisMode}
-                patientCount={this.props.store.filteredPatients.result!.length}
-            />
+            <>
```

Comment:
no biggie but I don't think you need the <> </> and also that semicolon down there is probably getting rendered.  should be aced

Reply:
i would just do 
```
if (this.props.store.filteredPatients.isComplete) {
   return <Lollipop...>
} else {
  return null;
}
```

Reply:
fixed